src/styles/month.scss

```diff
@@ -0,0 +1,65 @@
+@import 'variables';
+
+.calendar {
+  width: $cellsize * 7 + $gap * 6;
+  padding: 10px;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    background-color: #eee;
+    width: $cellsize;
+    height: $cellsize;
+    border: 1px solid black;
+    position: relative;
+    transition:
+      background-color 500ms,
+      transform 500ms;
+
+    &::before {
+      content: '';
+      display: flex;
```

Comment:
The `display: flex;` property is not necessary for pseudo-elements like `::before`. Consider removing it to avoid confusion.